### PR TITLE
SANDBOX-660: add webhook_url secret

### PR DESCRIPTION
**Pull request by rsoaresd**
# Description
As a requirement for onboarding Sandbox into ReportPortal, we must add a secret to OpenShift CI to store the Webhook URL to download and push data to ReportPortal. For more information, check [SANDBOX-438](https://issues.redhat.com/browse/SANDBOX-438).

## Issue ticket number and link
[SANDBOX-660](https://issues.redhat.com/browse/SANDBOX-660)

### CODE REVIEW

**Review by MatousJobanek — APPROVED** [review]
/lgtm

File: ci-operator/config/codeready-toolchain/host-operator/codeready-toolchain-host-operator-master.yaml [hunk 1]
```diff
@@ -26,6 +26,10 @@ tests:
     - as: test
       cli: latest
       commands: make test-e2e
+      credentials:
+      - mount_path: /usr/local/sandbox/ci-secrets/webhook_url
+        name: webhook_url
+        namespace: test-credentials
```
Comment (alexeykazakov) [inline ci-operator/config/codeready-toolchain/host-operator/codeready-toolchain-host-operator-master.yaml:32]:
```suggestion
        namespace: test-credentials
        
```

**Pull request merged**